pkg/git/git.go

```diff
@@ -116,6 +117,8 @@ func changeBranch(branchName string, worktree utilsWorkTree) error {
 // not reachable by 'from'.
 func LogRange(repo *git.Repository, from, to string) (object.CommitIter, error) {
 
+	rx := &RepositoryExt{Repo: repo}
```

Comment:
Is that something (`RepositoryExt`) something official? I can't find it in the go-git repo. Is it a PR to that repo somewhere? I personally would not use anything which is not yet merged into the sources. Who know if it reaches the sources at some later point in time.

In case this has been merged it would be - of course - good to use it instead of providing that from-to-iterator here ourselves.

Reply:
Nope, encapsulating the proposal of  https://github.com/Ajnasz/go-git/. I'll discard this commit.